book/forms.rst

```diff
@@ -1468,7 +1473,7 @@ renders the form:
 
         {# app/Resources/views/Default/new.html.twig #}
         {% form_theme form 'Form/fields.html.twig' %}
-
+        {# or if providing multiple themes #}
```

Comment:
Not sure about this one, but it might come in handy to inform that you use one of them depending on your situation.